packages/react-aria-components/docs/Autocomplete.mdx

```diff
@@ -116,6 +124,452 @@ import {UNSTABLE_Autocomplete as Autocomplete, SearchField, Menu} from 'react-ar
+
+```tsx example export=true
+import type {AutocompleteProps, MenuItemProps} from 'react-aria-components';
+import {Menu, MenuItem} from 'react-aria-components';
+
+interface MyAutocompleteProps<T extends object> extends Omit<AutocompleteProps, 'children'> {
+  label?: string,
+  placeholder?: string,
+  items?: Iterable<T>;
+  children: React.ReactNode | ((item: T) => React.ReactNode)
+  onAction?: (id: Key) => void
+}
+
+function MyAutocomplete<T extends object>({label,placeholder, items, children, onAction, ...props}: MyAutocompleteProps<T>) {
+  let {contains} = useFilter({sensitivity: 'base'});
+  return (
+    <div className="my-autocomplete">
+      <Autocomplete filter={contains} {...props}>
+        <MySearchField label={label} placeholder={placeholder} />
+        <Menu items={items} onAction={onAction}>
+          {children}
+        </Menu>
+      </Autocomplete>
+    </div>
+  );
+}
+
+function MyItem(props: MenuItemProps) {
+  return <MenuItem {...props} className={({isFocused, isSelected}) => `my-item ${isFocused ? 'focused' : ''} ${isSelected ? 'selected' : ''}`} />
+}
+
+<MyAutocomplete label="Commands" placeholder="Search commands...">
+  <MyItem>Create new file...</MyItem>
+  <MyItem>Create new folder...</MyItem>
+  <MyItem>Assign to...</MyItem>
+  <MyItem>Assign to me</MyItem>
+  <MyItem>Change status...</MyItem>
+  <MyItem>Change priority...</MyItem>
+  <MyItem>Add label...</MyItem>
+  <MyItem>Remove label...</MyItem>
+</MyAutocomplete>
+```
+
+<details>
+  <summary style={{fontWeight: 'bold'}}><ChevronRight size="S" /> Show CSS</summary>
+
+```css
+.my-autocomplete {
+  display: flex;
+  flex-direction: column;
+  gap: 12px;
+  max-width: 300px;
+  height: 180px;
+  border: 1px solid var(--border-color);
+  padding: 16px;
+  border-radius: 10px;
+  background: var(--overlay-background);
+}
+
+.react-aria-SearchField {
+  width: 100%;
+}
+
+.react-aria-Label {
+  margin-bottom: .5em;
+}
+
+.my-item {
+  margin: 2px;
+  padding: 4px 8px 4px 22px;
+  border-radius: 6px;
+  outline: none;
+  cursor: default;
+  color: var(--text-color);
+  font-size: 1.072rem;
+  position: relative;
+
+  &.selected {
+    font-weight: 600;
+    background: none;
+
+    &::before {
+      content: '✓';
+      content: '✓' / '';
+      alt: ' ';
+      position: absolute;
+      top: 4px;
+      left: 4px;
+    }
+  }
+
+  &.focused {
+    background: #e70073;
+    color: white;
+  }
+}
+
+@media (forced-colors: active) {
+  .my-item.focused {
+    background: Highlight;
+    color: HighlightText;
+  }
+}
+```
+
+</details>
+
+
+## Content
+
+Autocomplete follows the [Collection Components API](collections.html), accepting both static and dynamic collections.
+The examples above show static collections, which can be used when the full list of options is known ahead of time. Dynamic collections,
+as shown below, can be used when the options come from an external data source such as an API call, or update over time.
+
```

Comment:
this is really about menu/listbox rather than autocomplete itself... we may want to emphasize somehow that Autocomplete works with Menu or ListBox and this isn't specific to either one of those or autocomplete itself.

The Reusable Wrappers example also sort of suggests it requires a Menu. Not sure what the best way is to show that it works with both...